webui/master/src/store/index.tsx

```diff
@@ -22,9 +22,9 @@ import { ILogsState } from '@alluxio/common-ui/src/store/logs/types';
 import { browseReducer, initialBrowseState } from './browse/reducer';
 import { browseSaga } from './browse/sagas';
 import { IBrowseState } from './browse/types';
-import { configReducer, initialConfigState } from './config/reducer';
-import { configSaga } from './config/sagas';
-import { IConfigState } from './config/types';
+import { configReducer, initialConfigState } from '../../../common/src/store/config/reducer';
+import { configSaga } from '../../../common/src/store/config/sagas';
+import { IConfigState } from '../../../common/src/store/config/types';
```

Comment:
import from `@alluxio/common-ui/...` instead of `../../../` similar to lines 20-21.